compass/ocean/tests/ice_shelf_2d/viz.py

```diff
@@ -68,10 +83,80 @@ def run(self):
                                       dims=('yCell', 'nVertLevelsP1'))
         zInterface = zInterface.where(interfaceMask)
 
-        plt.figure(figsize=[24, 12], dpi=100)
-        plt.plot(ds.yCell.values, zMid.values, 'b')
-        plt.plot(ds.yCell.values, zInterface.values, 'k')
-        plt.plot(ds.yCell.values, ds.ssh.values, 'g')
-        plt.plot(ds.yCell.values, -ds.bottomDepth.values, 'g')
+        y = ds.yCell.values / 1.e3
+        plt.figure(figsize=[12, 6], dpi=100)
+        plt.plot(y, zMid.values, 'b', label='zMid')
+        plt.plot(y, zInterface.values, 'k', label='zTop')
+        plt.plot(y, ds.ssh.values, '--g', label='SSH')
+        plt.plot(y, -ds.bottomDepth.values, '--r', label='zBed')
+        plt.xlabel('Distance (km)')
+        plt.ylabel('Depth (m)')
+        plt.legend()
```

Comment:
Maybe drop the legend:
![vert_grid](https://github.com/MPAS-Dev/compass/assets/4179064/0d09af4c-f6c6-49fc-8269-2a3628c5b9cc)

Reply:
Or have it only for single layer?  Or the first layer of each type?